### PR TITLE
Added RUNNER_WORKDIR mounting readme note

### PR DESCRIPTION
**Pull request by EmpireJones**
Fix for https://github.com/myoung34/docker-github-actions-runner/issues/27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ This has been tested and verified on:
  * armv7
  * arm64
  
-**NOTE: Only one runner can use the same RUNNER_WORKDIR if it is shared storage.**
+**NOTE:**
+- **Only one runner can use the same RUNNER_WORKDIR if it is shared storage.**
+- **RUNNER_WORKDIR must match the source path for the bind-mounted volume at RUNNER_WORKDIR, in order for container actions to access files.**
```
Comment (myoung34) [inline README.md:16]: `- **RUNNER_WORKDIR must match the source path (and exist) for the bind-mounted volume at RUNNER_WORKDIR, in order for container actions to access files.**`

Comment (myoung34) [reply README.md:16]: i added `(and exist)` because that seemed like part of your finding also

**Review by myoung34 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EmpireJones) [reply README.md:16]: I think this shouldn't be a problem, per the docker docs:
> When the host directory of a bind-mounted volume doesn’t exist, Docker will automatically create this directory on the host for you.
- https://docs.docker.com/engine/reference/commandline/run/#mount-volume--v---read-only

What I meant by my original comment of "must exist on the host system" is that if the src and dest paths don't match, RUNNER_WORKDIR wouldn't exist on the host. If they do match though, it'll exist, per the docs mentioned above.

Comment (myoung34) [reply README.md:16]: OH. ok.

**Review by myoung34 — APPROVED** [review]

**Comment by myoung34** [comment]
Thanks!

**Pull request merged**